File: day_six/odd_or_even.py
#!/usr/bin/python3

import sys

def odd_or_even(num):

    if num % 2 == 0:
        print("It is even!")
    else:
        print("It is odd!")

def main():
    num = int(sys.argv[1])
    odd_or_even(num)

if __name__=="__main__":
    main()

File: day_six/oddOrEven.cpp
#include <iostream>

int main() 
{
    int num;

    std::cout << "Enter a num: \n";
    std::cin >> num;

    if (num % 2 == 0) 
    {
        std::cout << "It is even!\n";
    } 
    else 
    {
        std::cout << "It is odd!\n";
    }

    return 0;
}